jobs/yoyowallet_backend_platform_engineer.html

```diff
@@ -0,0 +1,60 @@
+---
+title: Backend Platform Engineer
+company: Yoyo Wallet
+url: https://boards.greenhouse.io/yoyowallet/jobs/570082
+location: London, England
+contract: permanent
+contact:
+    name: Jian Yuan Lee
+    email: jian@yoyowallet.com
+created: !!timestamp '2017-02-17'
+tags:
+  - python
+  - sql
+  - django
+  - postgresql
+  - rabbitmq
+  - celery
+  - docker
+  - aws
+---
+
+# Yoyo Wallet
```

Comment:
Somewhere in the ad you might want to include a "how to apply" section. You could consider listing out what you expect to see: e.g. CV or LinkedIn profile. GitHub samples. Do you want to see a cover letter. It helps to make these things clear if you have an idea of what you are after.